python/tests/test_aac/execute/test_aac_execution_result.py

```diff
@@ -0,0 +1,47 @@
+from unittest import TestCase
+from aac.execute.aac_execution_result import ExecutionResult, ExecutionMessage, ExecutionStatus, MessageLevel
+
+class TestExecitionResult(TestCase):
+
+    def test_execution_result(self):
+        execution_result = ExecutionResult("name", "command_name", ExecutionStatus.GENERAL_FAILURE, [])
+        execution_result.add_messages([
+            ExecutionMessage("message 1", MessageLevel.DEBUG, None, None),
+            ExecutionMessage("message 2", MessageLevel.INFO, None, None),
+            ExecutionMessage("message 3", MessageLevel.WARNING, None, None),
+            ExecutionMessage("message 4", MessageLevel.ERROR, None, None)
+            ])
+        execution_result.status_code = ExecutionStatus.SUCCESS
```

Comment:
And since you change the status here, can add a check prior for evaluating the full `is_success` logic.
```suggestion
        self.assertFalse(execution_result.is_success)
        
        execution_result.status_code = ExecutionStatus.SUCCESS
```